Moderation-Policy.md

```diff
@@ -305,10 +305,13 @@ remove resigning team member from respective permissions and private access.
 * [ovflowd](https://github.com/ovflowd) -
 **Claudio Wunder** &lt;cwunder@gnome.org&gt; (he/they)
 * [ryanmurakami](https://github.com/ryanmurakami) -
-**Ryan Lewis** &lt;ryanharrisonlewis@gmail.com&gt;
-* [Trott](https://github.com/Trott) -
 **Rich Trott** &lt;rtrott@gmail.com&gt;
-
+* [bmuenzenmeyer](https://github.com/bmuenzenmeyer) -
+**Brian Muenzenmeyer** <<brian.muenzenmeyer@gmail.com>> (he/him)
+* [UlisesGascon](https://github.com/ulisesgascon) -
+**Ulises Gascón** <<ulisesgascongonzalez@gmail.com>> (he/him)
+* [atlowChemi](https://github.com/atlowChemi) -
+**Chemi Atlow** <<chemi@atlow.co.il>> (he/him)
```

Comment:
Can you keep the list ordered by GH handle? We use case-insensitive alphabetical order usually.